### DISCUSSION TITLE
How do I use LaTeX intertext in quarto?

### DISCUSSION THREAD

**Discussion by justsharan** [quarto-dev/quarto-cli#9211] [Q&A]
### Description

Hey! I'm writing a statistics ebook in quarto, and I'd like to use `\intertext` within my align environments in LaTeX codeblocks to add text in-between. However, when I try to add it, this is what I see:

```qmd
$$
\begin{align*}
    M_X(t) &= \E\left(e^{tX}\right) \\
    \intertext{We can expand this using LOTUS.}
    &= \sum_{x=0}^\infty e^{tx} e^{-\lambda} \frac{\lambda^x}{x!}\\
    &= e^{-\lambda} \sum_{x=0}^\infty e^{tx} \frac{\lambda^x}{x!} \\
    &= e^{-\lambda} \sum_{x=0}^\infty \frac{\left(\lambda e^t\right)^x}{x!} \\
    \intertext{Due to the Maclaurin series of $e^x$ (check above), we can simplify the summation to $e^{\lambda e^t}$. The Maclaurin series of $e^x$ converges for all $x\in \mathbb{R}$, so it can be used here.}
    &= e^{-\lambda} e^{\lambda e^t}\\
    &= e^{\lambda e^t - \lambda} \\
    &= e^{\lambda (e^t-1)}
\end{align*}
$$
```
<img width="1236" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/53885408/dd417de0-0b17-4b1a-a10f-44b7331bde41">

Meanwhile, the expected behavior in a true LaTeX file would be:
<img width="744" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/53885408/a0496493-6be5-4ecb-ab01-c00d696252e9">

**Comment by mcanouil** [comment]
Use `keep-tex: true` in your YAML configuration file or YAML front matter.
Then look at the latex intermediary file and debug it.
Once you've found how to fix the issue, add that fix using `include-in-*` and/or LaTeX partials.

**Reply by justsharan** [reply]
I forgot to mention that I'm rendering the ebook as html, not pdf. I believe `keep-tex` may be a pdf-only option?

**Reply by cscheid** [reply]
> I forgot to mention that I'm rendering the ebook as html

In that case, you need to make sure that intertext is supported by MathJax (or whatever other `html-math-method` you might be using). That isn't really a thing that Quarto knows about.

**Reply by cderv** [reply]
> you need to make sure that intertext is supported by MathJax

And I don't think they have it 
* https://github.com/mathjax/MathJax/issues/736

@justsharan here is the doc about LaTeX equation in HTML document: 
https://quarto.org/docs/output-formats/html-basics.html#latex-equations

You have several options and what you are using is rendered by those. 

Hope it helps
